site/content/en/v1.3/install/install-yaml.md

```diff
@@ -38,9 +29,9 @@ Refer to the [Developer Guide](../../contributions/develop) to learn more.
 
    Envoy Gateway should now be successfully installed and running, but in order to experience more abilities of Envoy Gateway, you can refer to [Tasks](/latest/tasks).
 
-## Upgrading from v1.2
+## Upgrading from a previous version
```

Comment:
the actual previous version here matters, we dont garuntee upgrades for N-2 to N yet

Reply:
What about `the previous version`?  Then we don't have to keep updating the doc after each release.